Jenkinsfile

```diff
@@ -95,6 +95,27 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                             trap 'set -x; kill -INT \\\$AGENT_PID \\\$COPROC_PID' EXIT
                             OFI_INTERFACE=eth0 daos_test -m'''
 
+def rpm_scan_daos_test = '''pushd /tmp
+                              wget \
+                              http://rfxn.com/downloads/maldetect-current.tar.gz
+                              rm -rf /tmp/maldet-current
+                              mkdir -p /tmp/maldet-current
+                              tar -C /tmp/maldet-current --strip-components=1 \
+                                -xf maldetect-current.tar.gz
+                              cd maldet-current
+                              sudo ./install.sh
+                              sudo ln -s /usr/local/maldetect/ /bin/maldet
+                            popd
+                            sudo freshclam
+                            rm -f /tmp/clamscan.out
+                            sudo clamscan -d /usr/local/maldetect/sigs/rfxn.ndb \
```

Comment:
Is root really needed for this?

Reply:
Root is probably not needed for the scan.